### PR TITLE
Change from FTP to HTTP site

### PR DESCRIPTION
**Pull request by weiji14**
Point to https://generic-mapping-tools.iag.usp.br/gmt/gshhg-gmt-2.3.7.tar.gz instead of ftp://ftp.iag.usp.br/pub/gmt/gshhg-gmt-2.3.7.tar.gz. See also https://github.com/GenericMappingTools/gshhg-gmt and https://www.generic-mapping-tools.org/mirrors/index.html. Fixes timeout issue on https://github.com/conda-forge/gshhg-gmt-feedstock/pull/5.

**Note**: Will probably need to add `[ci skip] [skip ci] [cf admin skip] ***NO_CI***` to the commit title when merging this PR.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seisman** [comment]
Why not just use the github release link https://github.com/GenericMappingTools/gshhg-gmt/releases/download/2.3.7/gshhg-gmt-2.3.7.tar.gz?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: ftp://ftp.iag.usp.br/pub/gmt/{{ name }}-{{ version }}.tar.gz
+  url: https://generic-mapping-tools.iag.usp.br/gmt/{{ name }}-{{ version }}.tar.gz
```
[the remark this answers is not in the document]
Comment (seisman) [reply recipe/meta.yaml:11]: I believe we always use X.Y.Z instead of vX.Y.Z in gmt, dcw and gshhg repos.

[the remark this answers is not in the document]
Comment (weiji14) [reply recipe/meta.yaml:11]: Ok, I'll make the change then!

**Review by leouieda — APPROVED** [review]
Looks good to me. Might be best to bump the build number and let it build a new version just in case.

**Comment by weiji14** [comment]
> Looks good to me. Might be best to bump the build number and let it build a new version just in case.

Ok, done!

**Comment by weiji14** [comment]
@conda-forge-admin, please rerender

**Pull request merged**